conn.go

```diff
@@ -206,6 +206,7 @@ func (conn *Conn) buildPath(filename string) (fullPath string) {
 		fullPath = filepath.Clean(conn.namePrefix)
 	}
 	fullPath = strings.Replace(fullPath, "//", "/", -1)
+	fullPath = strings.Replace(fullPath, string(filepath.Separator), "/", -1)
```

Comment:
A better way of fixing the slash/backslash issue is to use the path.* functions rather than filepath. The path.* functions uses slash on all platforms.